src/main/java/org/opensearch/security/support/Base64Helper.java

```diff
@@ -220,7 +225,11 @@ private SafeObjectOutputStream(OutputStream out) throws IOException {
 
         @Override
         protected void writeClassDescriptor(final ObjectStreamClass desc) throws IOException {
```

Comment:
Can we add a tests that confirm this behavior?  It would be good to know at least that the rewrite function is not rewriting on a 1.3 cluster

Reply:
I duplicated the tests to both run both with and without the package rewrite.